raiden/api/python.py

```diff
@@ -370,6 +375,14 @@ def channel_open(
             )
 
         token_network = self.raiden.chain.token_network(registry.get_token_network(token_address))
+        given_block_identifier = views.state_from_raiden(self.raiden).block_hash
+
+        if self.channel_already_exists(
+            token_network_address=token_network_address,
+            partner_address=partner_address,
+            block_identifier=given_block_identifier,
+        ):
+            raise DuplicatedChannelError
```

Comment:
So the exception's name is self-explanatory but could you please add a message adding more details to the exception? like channel identifier / token_network address, partner address .. etc. The point is to have error messages that could help debugging this better.